graphics/image.h

```diff
@@ -277,6 +278,28 @@ class Image {
   bool DrawRectangle(int x, int y, int width, int height, int red, int green,
                      int blue);
 
+  /**
+   * Draws a polygon whose vertices are listed in |points| and colored with
+   * |color|. Each vertex is represented by it's x and y coordinate, and are
+   * listed sequentially in the vector. For example, a polygon with three
+   * vertices (0, 0), (0, 2), (2,1) is represented as a vector of integers
+   * {0, 0, 0, 2, 2, 1}. The last vertex will connect with the first vertex in
```

Comment:
It's a little hard to read this polygon vector, especially for large numbers of vertices. What do you think of making a simple graphics::Point class and having a std::vector<graphics::Point> instead?

You could also add a DrawTriangle that takes int x1, int y1, int x2, int y2, int x3, int y3 as that's probably the most common polygon that would be used.